docs/simplification.md

```diff
@@ -12,89 +18,83 @@ The AsyncAPI Model SDK supports simplification of:
 - [Required](#determining-the-required-properties-for-the-model)
 - [Extend](#determining-the-extend-for-the-model)
 
+<br/>
```

Comment:
I'm not sure this line break provides any improvement here. Also, if needed, you could add another empty line here as per markdown standards.